testcontainers/src/core/wait/mod.rs

```diff
@@ -13,6 +15,8 @@ use crate::{
 pub(crate) mod cmd_wait;
 pub(crate) mod exit_strategy;
 pub(crate) mod health_strategy;
+#[cfg(feature = "http_wait")]
+#[cfg_attr(docsrs, doc(cfg(feature = "http_wait")))]
```

Comment:
This is private definition, shouldn't be needed
```suggestion
```

Reply:
right, that makes sense, thanks